src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/repository/entity/UserRole.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.externalusersapi.repository.entity
+
+import org.springframework.data.relational.core.mapping.Table
+import java.util.UUID
+
+@Table(name = "user_role")
+data class UserRole(
+  val userId: UUID?,
+  val roleId: UUID?,
```

Comment:
Do they need to be non-null?

Reply:
Both should be mandatory while creating UserRole and UserGroup object